index.js

```diff
@@ -0,0 +1,126 @@
+var PARAMS = /\{([^\}]*)\}/g;
+var PROPS = /\{\{([^\}]*)\}\}/g;
+
+var crossroads = require('crossroads');
+var objectpath = require('object-path');
+var humanize = require('string-humanize');
+var path = require('path');
+var strip = require('strip');
+var urlRegex = require('url-regex');
+
+function parse( content, context ){
+  if( !content ) return false;
+
+  // Files
+  if( content.split('.').pop().match(/gif|jpg|jpeg|png/) ){
+    return content
```

Comment:
Good question! I'm thinking this was a lame attempt at error handling in the first pass. I've removed it, tests pass.